torch/ao/quantization/pt2e/_numeric_debugger.py

```diff
@@ -42,26 +43,50 @@ def generate_numeric_debug_handle(ep: ExportedProgram) -> None:
         )
 
     unique_id = 0
-    # Find the max ID that exists in the graph first, in case part of the graph
-    # has already been annotated. This way we guarantee there are no duplicate
-    # handle IDs.
-    for node in ep.graph.nodes:
+
+    def _bfs_trace_graph_with_node_process(node_op: Callable) -> None:
+        nonlocal ep
```

Comment:
nit: why not pass this as an argument?

Reply:
Just wanna make `_bfs_trace_graph_with_node_process` more generic

Reply:
looks like this takes an arg in the next PR